legislation-parameters.md

```diff
@@ -0,0 +1,109 @@
+# Legislation parameters
+
+## Definition
+
+The legislation parameters are stored in XML file in the directory `parameters` (see the [directory `parameters`  of OpenFisca-France](https://github.com/openfisca/openfisca-france/blob/master/openfisca_france/parameters)).
+
+These files contain parameters which can be simple parameters (`<CODE>` tags) or scales (`<BAREME>` tags).
+
+It is ordered as a tree defined by `<NODE>` tags.
+
+Here is a simple parameter sample:
+
+```xml
+<CODE code="taux_plein" description="taux plein de la CSG déductible" format="percent">
+  <VALUE deb="2005-01-01" fin="2014-12-31" valeur=".042" />
+  <VALUE deb="1998-01-01" fin="2004-12-31" valeur=".038" />
+  <VALUE deb="1997-01-01" fin="1997-12-31" valeur=".01" />
+</CODE>
+```
+
+## Parameters and Time
+
+Coding a parameter means defining a list of values, and for each value the date where it starts to hold.
+
+Example: the threshold of the [`decote`](https://legislation.openfisca.fr/parameters/impot_revenu.decote.seuil)
+
+```xml
+  <NODE code="decote" description="Décote" origin="openfisca">
+    <CODE code="seuil" description="Seuil de la décôte" format="integer" origin="openfisca" type="monetary">
+      <VALUE deb="2014-01-01" valeur="1016" />
+      <VALUE deb="2013-01-01" valeur="1016" />
+      <VALUE deb="2012-01-01" valeur="960" />
+      <VALUE deb="2010-01-01" valeur="878" />
+```
+
+### The tag `<END>`
+
+When a parameter is removed from the legislation, a tag `<END>` is used.
+
+Example: if `decote` is removed from legislation in 2017, the previous example would become
+
+```xml
+  <NODE code="decote" description="Décote" origin="openfisca">
+    <CODE code="seuil" description="Seuil de la décôte" format="integer" origin="openfisca" type="monetary">
+      <END deb="2017-01-01" />
+      <VALUE deb="2014-01-01" valeur="1016" />
+      <VALUE deb="2013-01-01" valeur="1016" />
+      <VALUE deb="2012-01-01" valeur="960" />
+      <VALUE deb="2010-01-01" valeur="878" />
+```
+
+### The tag `<PLACEHOLDER>`
+
+When a value is expected to change at a given date but the value is not published yet, a tag `<PLACEHOLDER>` is used.
+
+Example: if the threshold for `decote` is expected to change in 2018, the previous example would become
+
+```xml
+  <NODE code="decote" description="Décote" origin="openfisca">
+    <CODE code="seuil" description="Seuil de la décôte" format="integer" origin="openfisca" type="monetary">
+      <END deb="2017-01-01" />
+      <PLACEHOLDER deb="2018-01-01" />
+      <VALUE deb="2014-01-01" valeur="1016" />
+      <VALUE deb="2013-01-01" valeur="1016" />
+      <VALUE deb="2012-01-01" valeur="960" />
+      <VALUE deb="2010-01-01" valeur="878" />
+```
+
+The script [`find_placeholders.py`](https://github.com/openfisca/openfisca-core/tree/master/openfisca_core/scripts/find_placeholders.py) finds all placeholders in an legislation. It may be used periodically to alert on parameters about the change:
```

Comment:
- “in an legislation” -> “in a legislation”
- may -> can
- about the -> about to?